.gitignore

```diff
@@ -32,6 +32,7 @@ test-results
 .env
 /tests/installation/output/
 /tests/installation/.registry.json
+/tests/**/*.ts-snapshots/
```

Comment:
Is this change intentional? I don't see any directories with such names in my checkout.

Reply:
I can split this out if you want. I have a lot of these after running the screenshot tests as example:

```
	tests/page/page-screenshot.spec.ts-snapshots/page-screenshot-should-capture-css-transform-1-bidi-firefox-nightly.png
	tests/page/page-screenshot.spec.ts-snapshots/remove-should-work-bidi-firefox-nightly.png
```

Reply:
Oh, we cannot ignore. These are our expectations for each browser. We probably didn't commit expectations for bidi tests as they were failing, but I see at least some of them are there, e.g. `should-capture-css-box-shadow-1-bidi-chromium.png`. Instead of ignoring the directory, we should add missing bidi expectations there. In any case, let's keep it outside of this PR.

Reply:
Ok, I removed that particular commit.